community-members.md

```diff
@@ -45,23 +45,19 @@ Members of the Technical Committee are the maintainers of
 [Project specs](https://github.com/open-telemetry/opentelemetry-specification)
 and [Proto definitions](https://github.com/open-telemetry/opentelemetry-proto).
 
-Trace Approvers:
+[Specification sponsors](./community-membership.md#specification-sponsor) are
+trusted collaborators of the Technical Committee on Project specs and related
+repositories:
 
+- [Christian Beedgen](https://github.com/kumoroku), Sumo Logic
 - [Christian Neumüller](https://github.com/Oberon00), Dynatrace
-- [Nikita Salnikov-Tarnovski](https://github.com/iNikem), Splunk
-- [Ted Young](https://github.com/tedsuo), Lightstep
-
-Metrics Approvers:
-
 - [Cijo Thomas](https://github.com/cijothomas), Microsoft
-- [Leighton Chen](https://github.com/lzchen), Microsoft
-- [Tyler Yahn](https://github.com/MrAlias), Splunk
-
-Logs Approvers:
-
-- [Christian Beedgen](https://github.com/kumoroku), Sumo Logic
 - [Daniel Jaglowski](https://github.com/djaglowski), observIQ
 - [David Poncelow](https://github.com/zenmoto), Splunk
+- [Leighton Chen](https://github.com/lzchen), Microsoft
+- [Nikita Salnikov-Tarnovski](https://github.com/iNikem), Splunk
```

Comment:
@jack-berg Now that we're consolidating the sponsors, do you want to revisit the list in this PR or a separate PR? For example, @iNikem used to be very active couple years ago but not anymore, I don't know if he is still willing to be listed here or moved to Emeritus.

Reply:
I want to revisit in a separate PR. That way we can have a single group of emeritus sponsors, rather than trying to figure out how to represent emeritus for an approver concept which no longer exists.

Reply:
Bad merge conflict resolution. Fixed! 

> Have we confirmed with these approvers that they no longer want to be sponsors?

Some have confirmed, we're waiting on confirmation from others: https://github.com/open-telemetry/community/pull/2070#issuecomment-2117947968